packages/eas-cli/src/submissions/BaseSubmitter.ts

```diff
@@ -83,6 +97,16 @@ abstract class BaseSubmitter<SubmissionContext, SubmissionOptions> {
       throw new Error('This should never happen');
     }
   }
+
+  private async getProjectUrl(): Promise<string> {
```

Comment:
```suggestion
  private async getProjectUrlAsync(): Promise<string> {
```
We use `Async` suffix for async functions 😉